dist/onfido.min.js

```diff
@@ -1,2 +1,2 @@
n(0);Object.defineProperty(e,"__esModule",{value:!0}),e.isEmpty=e.findKey=e.omit=e.find=void 0,t=i(t),r=i(r),o=i(o);e.find=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},t=arguments.length>1?arguments[1]:void 0;return(0,o.default)((0,r.default)(e),function(n){return t(e[n],n)})};e.omit=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},n=arguments.length>1&&void 0!==arguments[1]?arguments[1]:[];return(0,r.default)(e).reduce(function(r,o){return(0,t.default)(n,o)||(r[o]=e[o]),r},{})};e.findKey=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},t=arguments.length>1?arguments[1]:void 0;return(0,o.default)((0,r.default)(e),function(n){return t(e[n],n)})};e.isEmpty=function(){return 0===(0,r.default)(arguments.length>0&&void 0!==arguments[0]?arguments[0]:{}).length}})},function(e,t){e.exports=function(e,t){var n=function(){};n.prototype=t.prototype,e.prototype=new n,e.prototype.constructor=e}},function(e,t,n){n(144);var r=n(2).Object;e.exports=function(e,t){return r.create(e,t)}},function(e,t,n){n(145);var r=n(2).Object;e.exports=function(e,t,n){return r.defineProperty(e,t,n)}},function(e,t,n){n(149),e.exports=n(2).Object.setPrototypeOf},function(e,t,n){n(89),n(88),n(150),n(151),e.exports=n(2).Symbol},function(e,t,n){n(50),n(62),e.exports=n(61).f("iterator")},function(e,t,n){var r=n(28),o=n(73),i=n(141);e.exports=function(e){return function(t,n,a){var s,c=r(t),u=o(c.length),l=i(a,u);if(e&&n!=n){for(;u>l;)if((s=c[l++])!=s)return!0}else for(;u>l;l++)if((e||l in c)&&c[l]===n)return e||l||0;return!e&&-1}}},function(e,t,n){var r=n(33),o=n(82),i=n(38),a=n(73),s=n(316);e.exports=function(e,t){var n=1==e,c=2==e,u=3==e,l=4==e,f=6==e,d=5==e||f,p=t||s;return function(t,s,h){for(var m,v,y=i(t),g=o(y),b=r(s,h,3),_=a(g.length),k=0,w=n?p(t,_):c?p(t,0):void 0;_>k;k++)if((d||k in g)&&(m=g[k],v=b(m,k,y),e))if(n)w[k]=v;else if(v)switch(e){case 3:return!0;case 5:return m;case 6:return k;case 2:w.push(m)}else if(l)return!1;return f?-1:u||l?l:w}}},function(e,t,n){var r=n(42),o=n(16)("toStringTag"),i="Arguments"==r(function(){return arguments}());e.exports=function(e){var t,n,a;return void 0===e?"Undefined":null===e?"Null":"string"==typeof(n=function(e,t){try{return e[t]}catch(e){}}(t=Object(e),o))?n:i?r(t):"Object"==(a=r(t))&&"function"==typeof t.callee?"Arguments":a}},function(e,t,n){var r=n(15).document;e.exports=r&&r.documentElement},function(e,t,n){var r=n(42);e.exports=Array.isArray||function(e){return"Array"==r(e)}},function(e,t){e.exports=function(e,t){return{value:t,done:!!e}}},function(e,t){e.exports={"absolute-center":"onfido-sdk-ui-Video-absolute-center",challenge:"onfido-sdk-ui-Video-challenge",challengeTitle:"onfido-sdk-ui-Video-challengeTitle",introCopy:"onfido-sdk-ui-Video-introCopy",introBullets:"onfido-sdk-ui-Video-introBullets",introBullet:"onfido-sdk-ui-Video-introBullet",bolder:"onfido-sdk-ui-Video-bolder",introIcon:"onfido-sdk-ui-Video-introIcon",speak_out_loudIcon:"onfido-sdk-
```

Comment:
any reason for changing the dist file? it takes history space, so it's better if it's only changed for releases.

Reply:
No reason, I'll remove it.